src/extension.ts

```diff
@@ -8,4 +8,5 @@ const integrator = new ExtensionStarter(vscode, fileSystem);
 
 export const activate = async (context: vscode.ExtensionContext) => integrator.start(context);
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Shall we use `no-empty-function: ["error", { "allow": ["arrowFunctions"] }]` as we have many of them, and they're legitimate use?

https://eslint.org/docs/latest/rules/no-empty-function#allow-arrowfunctions

Reply:
Fixed with `@typescript-eslint/no-empty-function`. ✅